src/biogeophys/SaturatedExcessRunoffMod.F90

```diff
@@ -334,6 +334,10 @@ subroutine ComputeFsatVic(bounds, num_hydrologyc, filter_hydrologyc, &
     ! Citation: Wood et al. 1992, "A land-surface hydrology parameterization with subgrid
     ! variability for general circulation models", JGR 97(D3), 2717-2728.
     !
+    ! This implementation gives a first-order approximation to saturated excess runoff.
+    ! For now we're not including the more exact analytical solution, or even a better
+    ! numerical approximation.
+    !
```

Comment:
**Comment from @martynpclark Sep 13, 2017**

We can add the analytical solution here.